app/main.py

```diff
@@ -1,8 +1,6 @@
 def copy_file(command: str) -> None:
-    command, file, new_file = command.split("")
-    if command == "cp":
+    command, file, new_file = command.split()
+    if command == "cp" and file != new_file:
         with open(file, "r") as source, open(new_file, "w") as target:
             data = source.read()
             target.write(data)
```

Comment:
Sorry, I didn't say it before, but you can remove the redundant variable in this case and make it in one line